src/main/java/org/kainos/ea/team2/cli/JobSpecificationResponse.java

```diff
@@ -0,0 +1,65 @@
+package org.kainos.ea.team2.cli;
+
+public class JobSpecificationResponse {
+
+    // attributes
+
+    /**
+     * Specification summary of job.
+     */
+    private String jobSpecification;
+
+    /**
+     * Link to full job spec on sharepoint.
+     */
+    private String sharepointLink;
+
```

Comment:
is there any other info we might want to display on the frontend screen? role name perhaps?